.bin/srclib-javascript

```diff
@@ -70,7 +70,6 @@ parser.command("scan-commonjs")
           if (deps) Object.keys(deps).forEach(function(name) { allDeps.push({name: name, version: deps[name]}); });
         }
         collectDeps(pkg.package.dependencies);
-        collectDeps(pkg.package.devDependencies);
```

Comment:
I think that's rare especially for code you depend on -- you might have a point for code in your own environment.  But w/ deps at least you're not going to be distributing tests or using them in actual code.

I think the npm convention is deps are for distribution and devDeps aren't, and you should only receive production-ready releases.  Note the NPM install `--production` flag: https://docs.npmjs.com/cli/install, it seems like if people are using `devDependencies` to include "actual dependencies", they're not using `devDependencies` correctly.

So if you're depending on a package, you don't care about their dev-deps, only their production-ready release.  Maybe there's a way we can provide a flag to srclib-javascript, so that if it's a local root project, we care about devDeps, but otherwise we ignore?

Also: what are your thoughts on peerdeps?